apps/docs/src/content/examples/postprocessing/ScreenQuad.mdx

```diff
@@ -4,36 +4,66 @@ This example demonstrates a well-known technique for doing simple postprocessing
 
-So the steps are as follows:
+The steps are as follows:
 
-1. render the "main" scene to the render target.
-2. render the separate scene.
+1. Render the "main" scene to the render target. The target's texture is passed as a uniform into the post-procssing shader.
+2. Render the scene that only contains the screen-quad mesh. This is what will be displayed on the canvas.
 
 There are some sub-steps inbetween but these are the two important bits that happen in the `useTask` callback in the `<Scene>` component.
 
 ## ScreenQuadGeometry
 
-This component creates a right triangle such that its right angle is positioned in the lower-left of the canvas. The size of the triangle is such that the hypotenuse only touches the top-right corner of the canvas. The triangle is in clip-space so we need a special vertex shader to use it.
+This component creates a buffer geometry triangle such that its right angle is positioned in the lower left of the canvas. The size of the triangle is such that the hypotenuse only touches the top right corner of the canvas. The triangle is in clip-space so we need a special vertex shader to use it.
 
 The vertex shader string is exported from the `<ScreenQuadGeometry>` component to be used with either a `Three.RawShaderMaterial` or `Three.ShaderMaterial`.
 
 ## Scene
 
 There are a few notable techniques in the `<Scene>` component.
 
-Firstly, The screen-quad mesh is not frustrum culled. This is because the geometry is so simple that culling it would actually requare more work. If you culled its geometry, you'd actually be creating more vertices.
+Firstly, The screen-quad mesh is not frustrum culled. This is because the geometry is always visible, so no point in culling it.
 
 This `<T.Mesh>` uses threlte's `attach` prop to add it to the secondary scene instead of the default scene created by threlte.
 
 Secondly, the render task that threlte automatically uses is disabled and a custom render task is used instead. This is explained on the [`useTask`](/docs/reference/core/use-task) page.
 
-Lastly, the uvs are used in the fragment shader to use as an index into the scene texture. For this reason, the resolution of the canvas is sent into the shader as a uniform. An effect is ran to keep the uniform in sync with the current size of the canvas.
+Lastly, the uvs are used in the fragment shader as an index into the render target's texture. Note that you can get a perfect circle by sending the size of the renderer into the fragment shader as a uniform.
+
+```ts
+import { fromStore } from 'svelte/store'
+
+const fragmentShader = `
+	// ...
+	uniform vec2 uSize;
+
+	vec2 uv = gl_FragCoord.xy / uSize.xy;
+	// ...
+`
+
+const uSize = new Uniform(new Vector2())
+
+const { size } = useThrelte()
+
+const sizeAsState = fromStore(size)
+
+$effect(() => {
+  uSize.value.set(sizeAsState.current.width, sizeAsState.current.height)
+})
```

Comment:
The `size` variable from useThrelte is a [currentReadable](https://github.com/threlte/threlte/blob/780b3c518fd6aac000ea79323cf4e01041a53095/packages/core/src/lib/context/fragments/dom.ts#L8C2-L8C59) so could use as is